src/ui.c

```diff
@@ -170,7 +170,7 @@ file_finder_draw(struct file_finder *finder)
 
 	wmove(finder->win, finder->height - 1, 0);
 	wbkgdset(finder->win, get_line_attr(NULL, LINE_TITLE_FOCUS));
-	wprintw(finder->win, "[finder] file %d of %d", pos->lineno + 1, finder->lines);
+	wprintw(finder->win, "[finder] file %lu of %zu", pos->lineno + 1, finder->lines);
```

Comment:
In https://github.com/jonas/tig/pull/1085/files#diff-662ec1e8c57906cca33cf41b09c520a818a03e8c7454538e3529869b8c2b24b4R693 you're using `%u` for `line->lineno`. Do the `lineno` members have different types?

Reply:
Yes, I noticed that as too, lineno is unsigned int in line struct while it is unsigned long in position struct. For now, I only wanted to restore the concordance between formats and types, but I agree it could deserve attention as well.